scale-tests/streaming_test.py

```diff
@@ -10,6 +10,8 @@
 
 Options:
     --desired-runtime <n>        desired time for consumers to run (in minutes) [default: 10]
+    --docker-image <image>       docker image to submit with spark job [default: mesosphere/spark:latest]
```

Comment:
I think you can omit this config property altogether. I added this originally to try various images. But, now that we add a stub universe repo during testing, the dispatcher and drivers will automatically inherit whatever image is in that stub universe (without having to set spark.executor.docker.image explicitly).

Reply:
sounds good i'll take it out.. FYI it doesn't seem like this script will be used too often now that we have the more robust kafka-c* streaming app

Reply:
but this is good to have if we ever want to kick off a bunch of simple kafka jobs that don't require quota or a backend